### PR TITLE
Add tinycc

### PR DESCRIPTION
**Pull request by hidalgopl**

### CODE REVIEW

File: pyshorteners/shorteners/__init__.py [hunk 1]
```diff
@@ -111,6 +113,7 @@ def short(self, url):
             self.kwargs['timeout'] = 0.5
 
         self.shorten = self._class(**self.kwargs).short(url)
+        print(self.shorten)
```
Comment (ellisonleao) [inline pyshorteners/shorteners/__init__.py]: can you remove this print statement, please?

**Comment by hidalgopl** [comment]
Hey @ellisonleao I've added the qr_code() method and a test for it ;)

File: pyshorteners/shorteners/__init__.py [hunk 2]
```diff
@@ -136,6 +135,8 @@ def expand(self, url=None):
     def qrcode(self, width=120, height=120):
         if not self.shorten:
             return None
+        if self.engine == Shorteners.TINYCC:
```
Comment (ellisonleao) [inline pyshorteners/shorteners/__init__.py]: you can implement the `qrcode` method inside the `Tinycc` class on `tinycc.py` file. Can you remove this if statement please?

**Comment by hidalgopl** [comment]
Hey @ellisonleao , I moved qrcode to Tinycc class ;)

**Pull request merged**